### DISCUSSION TITLE
git cherry-pick

### DISCUSSION THREAD

**Discussion by mehtapinal** [gitpython-developers/GitPython#1147] [Q&A] [answered]
what is the best way to use cherrypick using gitPython. I could not find this in the doc as well

**Comment by Byron** [comment, marked as answer]
I think you [could use git directly](https://gitpython.readthedocs.io/en/latest/tutorial.html#using-git-directly).
Hope that helps.
Please feel free to comment on the closed issue.

**Comment by mehtapinal** [comment]
Ok thanks!
